src/components/FlashWord/FlashWord.module.scss

```diff
@@ -1,6 +1,11 @@
 @import "../../styles/variables";
 
 .word {
+  position: absolute;
```

Comment:
Also why position absolute?

Reply:
I feel like these are questions, not requests for changes. Setting the position of these allows them to overlap. If they are static they will be arranged one after the other. If one is static, and the other relative (then relatively placed over the other) that isn't as responsive or stable (or my intention for that matter). To set it absolutely is telling it to be placed in the center of the div, even though there is another div overlapping it. Does that make sense?